tinygrad/renderer/triton.py

```diff
@@ -41,8 +41,10 @@ def define_scalar(local_size, dtype, args):
 
 def uops_to_triton(function_name:str, uops:List[UOp]):
   local_size: List[int] = []
+  dims: List[str] = []
```

Comment:
Inferred this type from the data put onto the list on 106. 107 - 112 indicates the type is string.